.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
```

Comment:
Was this file related to the upgrade, or added separately?

Reply:
This file was generated by `npx d2-style install project/react --force` see: https://cli-style.dhis2.nu/#/migrate-guide